File: LeetCode-Programs/Maximum Number of Words Found in Sentences.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        max_word=0
        for i in range(len(sentences)):
            count=1
            word=sentences[i]
            for j in range(len(word)):
                if sentences[i][j]==' ':
                    count+=1
                max_word=max(max_word, count)
        return max_word

File: LeetCode-Programs/Maximum Number of Words Found in Sentences.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int max_words=0;
        for(int i=0;i<sentences.size();i++){
            int count=1;
            for(int j=0;j<sentences[i].size();j++){
                if(sentences[i][j]==' ')
                    count++;
            }
            max_words=max(max_words, count);
        }
        return max_words;
        
    }
};
